sdk/communication/Azure.Communication.CallingServer/src/CallConnection.cs

```diff
@@ -82,19 +84,30 @@ public virtual Response Hangup(CancellationToken cancellationToken = default)
             }
         }
 
-        /// <summary> Terminates the conversation for all participants in the call. </summary>
+        /// <summary> Disconnect the current caller in a group-call or end a p2p-call.</summary>
+        /// <param name="forEveryone"> If true, this will terminate the call and hang up on all participants in this call. </param>
         /// <param name="cancellationToken"> The cancellation token. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual async Task<Response> TerminateCallAsync(CancellationToken cancellationToken = default)
+        public virtual async Task<Response> HangupAsync(bool forEveryone, CancellationToken cancellationToken = default)
```

Comment:
Should forEveryone have a default of false?

Reply:
I guess it depends on how frequently we expect these methods to be used as well.  If majority of time the user would just hangup their own callConnection, I feel having default value would be useful so we don't always have to repeatedly pass in the same value, except in exceptional case where you want to actually terminate a call.  If we expect fairly equal usage for both hangup/terminate, then mandatory property makes most sense to me.

Reply:
I still kind of like keeping this mandatory, but I can bring this topic up during the meeting as one of the discussion topics